### PR TITLE
Add opencontainers image-spec to `Dockerfile`

### PR DESCRIPTION
**Pull request by SauravMaheshkar**
Congratulations on the release !! 🥳 🍾 

This PR makes a few tiny changes to improve the overall quality of the docker image 🐳 

---

For reference more annotations can be found [here](https://github.com/opencontainers/image-spec/blob/main/annotations.md)

### CODE REVIEW

**Comment by SauravMaheshkar** [comment]
Request for Review: @decarboxy @gahdritz

**Comment by gahdritz** [comment]
Could you summarize what this does?

**Comment by SauravMaheshkar** [comment]
Sure, Opencontainers is a open governance structure dedicated to creating open standards around container technology and has defined a "open-review" image annotations that can be added to Images to improve the overall quality of the `Dockerfile`.

The purpose of this PR is to add said annotations that make the Image follow these said formal specifications.

They can be thought of as metadata for the Dockerfile that have no effect on the Image but at the same time provides extra context.

**Comment by SauravMaheshkar** [comment]
Thoughts @gahdritz?
